service/service.go

```diff
@@ -56,9 +57,19 @@ func Run(ctx context.Context, cfg *config.Config, serviceList *ExternalServiceLi
 		return nil, errors.Wrap(err, "unable to register checkers")
 	}
 
+	var homepageClient homepage.HomepageClienter
+	if cfg.IsPublishingMode {
+		homepageClient = homepage.NewHomePagePublishingClient(svc.clients)
+	} else {
+		languages := strings.Split(cfg.Languages, ",")
+		homepageClient = homepage.NewHomePageWebClient(svc.clients, cfg.CacheUpdateInterval, languages)
+		go homepageClient.StartBackgroundUpdate(ctx, svcErrors)
```

Comment:
Is `StartBackgroundUpdate` only needed in web mode?

Reply:
Its only needed in the web mode.